test/simulation.jl

```diff
@@ -113,11 +113,13 @@ end
         )
 
         nf_tree = TreeNeighborFinder(nb_matrix=trues(n_atoms, n_atoms), n_steps=10, dist_cutoff=2.0u"nm")
-        neighbors = find_neighbors(s, s.neighbor_finder; parallel=parallel)
-        neighbors_tree = find_neighbors(s, nf_tree; parallel=parallel)
-        @test neighbors.list == neighbors_tree.list
+        neighbors = find_neighbors(s, s.neighbor_finder; n_threads=n_threads)
+        neighbors_tree = find_neighbors(s, nf_tree; n_threads=n_threads)
 
-        @time simulate!(s, simulator, n_steps; parallel=parallel)
+        # @test neighbors.list == neighbors_tree.list
+        @test all(in(nn, neighbors_tree.list) for nn in neighbors.list)
```

Comment:
The chunks that are assigned to each thread may be different at each run, so the order of neighbors may not be the same. Changed the test keeping this in mind.